crates/cxx-qt-build/src/lib.rs

```diff
@@ -1167,6 +1167,13 @@ extern "C" bool {init_fun}() {{
 
         let mut qtbuild = qt_build_utils::QtBuild::new(qt_modules.iter().cloned().collect())
             .expect("Could not find Qt installation");
+        if let Some(autorcc) = env::var_os("CXX_QT_AUTORCC_OPTIONS") {
+            let autorcc = autorcc
+                .to_str()
+                .expect("CXX_QT_AUTORCC_OPTIONS are not UTF-8!");
+            let autorcc: Vec<_> = autorcc.split(':').collect();
```

Comment:
guess empty strings / values need to be filtered somewhere, either here or on the qt-build-utils side

Reply:
Added it on the cmake side.